### ISSUE TITLE
Add 'clear' command to clear terminal.

### ISSUE THREAD

**Issue by jghibiki** [moe-serifu-circle/moe-serifu-agent#29]
I was playing with it and the terminal gets very cluttered very quickly, and my eyes are bad. A clear command might be helpful for increasing readability of long terminal sessions.

[For reference here is an article (to save you time) on different ways to clear the screen in cpp.](http://www.cplusplus.com/articles/4z18T05o/)

**Comment by dekarrin** [comment]
@jghibiki I agree with your points on this, #30, and #31 but I won't be able to get to them for some time yet as I am currently occupied with creating the [moe-serifu-circle/website](https://github.com/moe-serifu-circle/website).

If you're interested in seeing these changes before then you totally have the go-ahead to make it so. I'd imagine the majority of the work would be done in src/output/output.cpp
